src/main/java/org/openstreetmap/atlas/checks/validation/linear/lines/GeneralizedCoastlineCheck.java

```diff
@@ -90,7 +92,8 @@ public boolean validCheckForObject(final AtlasObject object)
             // Contains midpoints and sharp angle locations
             final List<Location> pointsForFlagging = this.getPointsForFlagging((LineItem) object);
             // If there were sharp angles
-            if (pointsForFlagging.addAll(this.getSharpAngleLocations((LineItem) object)))
+            if (pointsForFlagging.addAll(this.getSharpAngleLocations((LineItem) object))
+                    && this.sharpAngleThreshold.asDegrees() > DO_NOT_EVALUATE_DEFAULT_CAP)
```

Comment:
Just a thought, I am not sure if this makes sense for this check or not. You could use `DO_NOT_EVALUATE_DEFAULT_CAP` as the default value for sharpAngleThreshold. That way if `angle.minimum.threshold` is not given in the config this functions is turned off, instead of needing to know that -1 needs to be given to switch it off. The disadvantage would be this feature is turned off by default.

Reply:
Good idea. I like this better since there might've been some confusion around valid configuration values. Made the changes in 3608a46